project.clj

```diff
@@ -9,6 +9,7 @@
 
   :dependencies
   [[manifold "0.1.8"]
+   [mvxcvi/multihash "2.0.3"]
```

Comment:
This project will be archived soonish, so for future-proofing I'd recommend switching to [mvxcvi/multiformats](https://github.com/greglook/clj-multiformats/). Alternately, you could internalize the hash calculation since you don't really need the self-describing nature of these hashes.